src/odemis/driver/technolution.py

```diff
@@ -773,10 +914,11 @@ def __init__(self, name, role, parent, **kwargs):
         self._scanner = self.parent._ebeam_scanner
         self._descanner = self.parent._mirror_descanner
 
-        self._shape = (8, 8, 65536)
+        self._shape = MPPC.SHAPE
```

Comment:
I don't see where `MPPC.SHAPE` is defined. It should be just above, just before the `MPPC.__init__`

Reply:
That is really sharp, somehow I didn't cherry pick the latest and full commit.
I will overwrite this commit with the full version which has as extra changes:


Added:
> SHAPE = (8, 8, 65536)


Changed:
In in settting externalStorageURL (self.externalStorageURL = model.StringVA)
`  "directory": "directory"}  -->    "directory": "image_dir"}`

and

In unitTest EbeamScanner test_resolution_VA
OLD:
```
# Check if for requested resolution values, where the effective cell size is not a multiple of 4, the closest
# correct resolution is returned
```
NEW:
```
# Check if for a requested resolution values, where the effective cell size is not a multiple of 4, the closest
# correct resolution is returned
```

Reply:
If you agree with these changes can you resolve this conversation?

Reply:
I only see the first change in your new commit. That said, the changes to the unit tests that you're describing look fine.

Reply:
Those changes where apparently still in a previous commit which I didn't cherry-picked. It is all solved now.